_config.yml

```diff
@@ -1,6 +1,13 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely need to edit after that.
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'jekyll serve'. If you change this file, please restart the server process.
+
 # Site settings
 title: I'll Acccept Anything
-email: your-email@domain.com
+email: progval+github@progval.net
```

Comment:
It's just a config file. I put it there because this is your repo. I can change it to.... my email if you want. LOL.